istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -334,7 +334,7 @@ func clusterConfigCmd(ctx cli.Context) *cobra.Command {
 	var podName, podNamespace string
 
 	clusterConfigCmd := &cobra.Command{
-		Use:   "cluster [<type>/]<name>[.<namespace>]",
+		Use:   "clusters [<type>/]<name>[.<namespace>]",
```

Comment:
+1, they are aliases that both should work.

Reply:
Hi @ymesika and @hanxiaop , thank you for your review, should I change back to "cluster" or leave it as it is?

Reply:
I think you need to change it back to `cluster` or you have to modify the aliases as well otherwise there will be a regression - the later one is too complicated so I prefer to change it back. If you see the tests, the failures are due to this modification.

Reply:
thanks @hanxiaop , I changed it back per your suggestion.